lib/services/profiles-api-service.js

```diff
@@ -315,11 +326,12 @@ function profileApiServiceFactory(
                                 'Unable to retrieve valid node bootSettings', node.type);
                     }
                 } else {
-                    return self._handleProfileRenderError(
-                        'Unable to locate active workflow or there are no bootSettings',
-                        node.type,
-                        400
-                    );
+                    return {
+                        profile: 'ipxe-info.ipxe',
```

Comment:
I think this could end up sending an ipxe script to a switch? Or maybe this code path already had that problem?